code/lib/core-server/src/dev-server.ts

```diff
@@ -37,6 +38,13 @@ export async function storybookDevServer(options: Options) {
     getServerChannel(server)
   );
 
+  if (features?.storyStoreV7 === false) {
+    deprecate(
+      dedent`storyStoreV6 is deprecated, please migrate to storyStoreV7 instead.
+      - Refer to the migration guide at https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#storystorev6-and-storiesof-is-deprecated`
+    );
```

Comment:
@yannbf I'd like to hear your opinion on if we should continue to do deprecations warnings like this -in place-?

I'd love to figure out a way to use the errors-consolidation work you've done, to consolidate deprecation warnings like these.

Reply:
let's create a task for this!